GameData/WarpPlugin/Patches/NFTEnginesFix.cfg

```diff
@@ -1,92 +1,3 @@
-@PART[ionXenon-*]:NEEDS[!NearFutureElectrical,!SETI,NearFuturePropulsion]:FOR[WarpPlugin]
```

Comment:
Just my preference - this patch breaks the ability to use NFT electrical engines with NFT solar panels which I sometimes use.